### ISSUE TITLE
Additional documentation added to bulleted lists

### ISSUE THREAD

**Issue by kfink24** [department-of-veterans-affairs/vets-design-system-documentation#2289]
### Duplicate check

- [X] I've searched for any related issues and avoided creating a duplicate issue.

### This update is for:

Component

### What is the name?

Bulleted Lists

### What is the nature of this update?

- [ ] How to build this component/pattern
- [X] When to use this component/pattern
- [ ] When to use something else
- [ ] Usability guidance
- [X] Accessibility
- [ ] Implementation
- [ ] Research insights
- [ ] Package information
- [ ] Addition to Forms Library documentation
- [ ] Update to existing Forms Library documentation

### What problem does this solve?

The current documentation on bulleted lists does not make it explicitly clear that you should use a ■ (square) vs. a •  (circle) for the bullet points.  In fact, the documentation guidance at the top of the page is using • circle bullets and this can cause confusion for teams that either option is available to use.  

Recommend adding documentation to clarify that only ■ should be used for the bulleted list items to prevent potentially harmful user experience to some veterans (ie. those using screen readers). 

### Additional Context

Screen readers read out • as "bullet" which could be triggering language for some veterans.  We should ensure that  our documentation is thorough so that there is not an accidental use of a •  which could lead to this triggering language.  Screen readers read out ■ as "black square" which is acceptable and should not cause issues for a veteran.

**Comment by humancompanion-usds** [comment]
Some discovery has been done on this topic. The notion of bullet being triggering sounds like it was based mainly on folks' opinions and not research. Thus Danielle is not comfortable putting this out there as content guidance. We are always aiming to make our language as absolutely empathetic as we can possibly make it. But we need to be careful to not make assumptions and especially not to veer into being patronizing. 

We did discover that for screen reader users the list items are being read out as "black square" because of the css list-style-type: square property. If we change to disc it will read as "bullet". 

Claudia (a blinded Veteran, one of our valued BVA members and active participants) says: 
> “I prefer the bullets, I just now learned that those black boxes are bullets, I have that on a form and I’m like what the heck is this black box? So I didn’t even know that.”

Thus the content clarity of a "bullet" in this context may outweigh any worries about that language being triggering. 

While this was being discussed I did add [List](https://design.va.gov/components/list) as a component and indicated that we currently use a black square instead of bullet. I also updated the Design System to use that list item style. Thus I'm going to close this issue because we have addressed the documentation gap. I'm going to open a new issue to globally switch the list items back to disc/bullet list-style-type and put that on our backlog of items to come to Design System Council at some point.
